### PR TITLE
Tidy up of PG & RAC for Aasimar and related feats and Regions

### PR DESCRIPTION
**Pull request by FerretDave**
Disabled a few deprecated references to Aasimar, corrected feat/template references to new Planetouched(xx) keys and added sortkey to regions

### CODE REVIEW

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/players_guide_to_faerun/pg_templates.lst [hunk 1]
```diff
@@ -102,7 +102,7 @@ Greater Power ECL Boost					KEY:Feyri ~ Greater Power ECL Boost		VISIBLE:NO
 
 
 ###Block:
-Lesser Planetouched		VISIBLE:YES		BONUS:VAR|DefaultRaceType|1	ABILITY:Internal|AUTOMATIC|Assign Race Type ~ Humanoid	TYPE:Humanoid	SUBRACE:Lesser	BONUS:VAR|LevelAdjustment|-1
+Lesser Planetouched		VISIBLE:YES		BONUS:VAR|DefaultRaceType|1	ABILITY:Internal|AUTOMATIC|Assign Race Type ~ Humanoid	TYPE:Humanoid	SUBRACE:Lesser	PRERACE:1,Planetouched (Aasimar),Planetouched (Tiefling)	BONUS:VAR|LevelAdjustment|-1
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/players_guide_to_faerun/pg_templates.lst]: I have Lesser Genasi coded up as well, should they be added to the PRERACE?

Comment (FerretDave) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/players_guide_to_faerun/pg_templates.lst]: I guess yes, I'll add those now. Have just fixed Turn vars on Cloistered
Cleric so will re-commit shortly.

**Comment by FerretDave** [comment]
updated to add Genasi reference, and finalise cloistered cleric

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/players_guide_to_faerun/pg_templates.lst (hunk 1 above)
Comment (FerretDave) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/players_guide_to_faerun/pg_templates.lst]: There, that should be it now.

**Comment by FerretDave** [comment]
Thank you, sending another one through too now.

Got one for you though - The Lesser Planetouched should not get the
benefits of being an outsider - so lose Darkvision and martial weapon profs.
Looks like they already do lose the profs, but I can't see quite how to
lose darkvision.
Not a problem for me right now, as the character has taken a level of
acolyte of the skin, which gives darkvision anyway... so can you check on
that please?

Cheers
Dave

**Pull request merged**